co-simulation/fed/mosaic-infrastructure/src/main/java/org/eclipse/mosaic/fed/infrastructure/ambassador/InfrastructureInstanceManager.java

```diff
@@ -94,6 +93,7 @@ private void newInfrastructureInstance(String infrastructureId, InetAddress rxMe
                 timeSyncPort, location);
         try {
             tmp.bind();
+            log.info("New infrastructure registration received with ID '{}'", infrastructureId);
```

Comment:
This is a sort of deceiving log statement. To me, it suggests that the CARMASimulation instance just received a new registration message but really this is only triggered after a registration message has been received and the next time step is executed. Better wording maybe "New Infrastructure instance {} registered with Infrastructure Instance Manager." or something. This way we can add a log statement to the actual InfrastructureRegistrationReceiver to log when the registration is received and then separately log when we register the instance. Does that make sense? Eric and I spent some time trying to figure out why we could not trigger this log statement.